testgrid/cmd/entomologist/main_test.go

```diff
@@ -58,10 +72,38 @@ func TestOptions(t *testing.T) {
 			expected: &options{
 				organization:   "testorg",
 				repository:     "testrepo",
-				gcsPath:        "gs://foo/bar",
+				output:         "gs://foo/bar",
 				gcsCredentials: "/usr/foo/creds.json",
 			},
 		},
+		{
+			name: "required options with directory output",
```

Comment:
I can see this script writing to a file, but I'm not sure how it would successfully write data to a directory. Can you explain your thinking here?